.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    strategy:
```

Comment:
You can set `fail-fast: false` to continue one release even if the other fails.

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast

Reply:
Good idea. Done.